tmk_core/protocol/lufa/lufa.c

```diff
@@ -517,17 +517,20 @@ void EVENT_USB_Device_ControlRequest(void)
                         if (USB_DeviceState == DEVICE_STATE_Unattached)
                           return;
                     }
-#ifdef KEYBOARD_SHARED_EP
-                    uint8_t report_id = REPORT_ID_KEYBOARD;
-                    if (keyboard_protocol) {
-                       report_id = Endpoint_Read_8();
-                    }
-                    if (report_id == REPORT_ID_KEYBOARD || report_id == REPORT_ID_NKRO) {
-                        keyboard_led_stats = Endpoint_Read_8();
+
+                    if (Endpoint_BytesInEndpoint() == 2) {
+                      uint8_t report_id = REPORT_ID_KEYBOARD;
+
+                      if (keyboard_protocol) {
+                        report_id = Endpoint_Read_8();
+                      }
+
+                      if (report_id == REPORT_ID_KEYBOARD || report_id == REPORT_ID_NKRO) {
+                          keyboard_led_stats = Endpoint_Read_8();
```

Comment:
```suggestion
                        keyboard_led_stats = Endpoint_Read_8();
```
Indentation. I'd also kill the two empty lines above, it's readable enough without them.

Reply:
I prefer to have empty lines between consecutive block statements and before keywords like break and return, personally I think that is more readable. I wish QMK had a more defined code style :(

Reply:
@fauxpark Open an issue/PR to improve it, please! :)
https://docs.qmk.fm/#/contributing?id=coding-conventions